### PR TITLE
(docs)(MODULES-2568) Fix and clean up Readme headings

### PR DESCRIPTION
**Pull request by jpogran**
Add the sub headings in the Usage section that are not present in the TOC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,9 +41,9 @@ puppet module install puppetlabs-dsc
 
 ## Usage
 
-### Refresh Mode Must be Disabled
+### LCM RefreshMode Must be Disabled
```
Comment (Iristyle) [inline README.md]: So I got word today that this is no longer a limitation based on the production preview bits.  @jpogran / @ferventcoder / @cowofevil want to verify that?

Comment (Iristyle) [inline README.md]: So I just verified locally - no need to touch the LCM anymore!  Submitting a PR to remove the PowerShell checks / update tests.

Comment (Iristyle) [inline README.md]: PR up at https://github.com/puppetlabs/puppetlabs-dsc/pull/98

Comment (ferventcoder) [inline README.md]: I was going to ask the same thing!

**Comment by ferventcoder** [comment]
I'm good to merge this. @Iristyle do you want to rebase #98 to remove this section of the readme?

**Pull request merged**